rqt_image_overlay/test/test_image_manager.cpp

```diff
@@ -94,3 +94,18 @@ TEST_F(TestImageManager, getHeaderTime)
   EXPECT_EQ(headerTime, time);
   EXPECT_NE(image, nullptr);
 }
+
```

Comment:
Thanks! I didn't check for tests. Next time I'll add one

Reply:
All good, as long as we can confirm the intended behviour :+1: